packages/client/src/components/Pagination/Pagination.js

```diff
@@ -12,26 +12,22 @@ function Pagination({ currentPage, pageCount, onPageChange }) {
   }
 
   const nextPage = () => {
-    if (currentPage === pageCount) {
-      onPageChange(1);
-    } else {
+    if (currentPage !== pageCount) {
```

Comment:
you can use navigate to change the url dynamically by changing the page 

navigate(`?page=${currentPage + 1}`);

Reply:
I will try to explore useNavigation after graduation. Now I rather to be cautious  than curious.